### PR TITLE
Add ability to add articles to Pocket

### PR DESCRIPTION
**Pull request by geoffreylitt**
I used your library in my own project, and needed the ability to add articles. I added a `pocket add <url>` command to the CLI as well.

Let me know if this change looks good to you, or if you'd like me to make modifications.

ちなみに、もし簡単であれば、日本語でも会話できます。

### CODE REVIEW

**Comment by geoffreylitt** [comment]
@motemen I just fixed some minor comment and formatting issues. Still wondering if you're interested in merging this change!

File: cmd/pocket/main.go [hunk 1]
```diff
@@ -142,6 +149,32 @@ func commandArchive(arguments map[string]interface{}, client *api.Client) {
...
+	if tags, ok := arguments["--tags"].(string); ok {
+		options.Tags = tags
+	}
+
+	err := client.Add(options)
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Println("Successfully added.")
```
Comment (motemen) [inline cmd/pocket/main.go:175]: I'd like have this part:
- if `err != nil`, print it out to stderr and exit with nonzero
- If `err == nil`, print nothing and exit with zero

If you don't mind, I myself can modify this part and merge this PR :smile:

File: api/add.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+type addAPIOptionWithAuth struct {
+	*AddOption
+	authInfo
+}
+
+type AddResult struct{}
+
+// Only returns an error status, since adding an article doesn't have
```
Comment (motemen) [inline api/add.go]: Can you start this doc with "Add ..." as [golint](https://github.com/golang/lint) complains? (Though other part of this project does not satisfy golint :stuck_out_tongue_winking_eye: )

**Comment by motemen** [comment]
Sorry for the late reply, mostly looks good and added some comments!

File: api/add.go (hunk 2 above)
Comment (geoffreylitt) [inline api/add.go]: Yep, fixed that so `golint` passes now.

File: cmd/pocket/main.go (hunk 1 above)
Comment (geoffreylitt) [inline cmd/pocket/main.go:175]: Sure, you can change that and merge! The rest of this file uses `panic`, do you want to change those as well? (I'm new to golang so I don't really know the difference)

Comment (motemen) [inline cmd/pocket/main.go:175]: Thanks! For now, other parts don't need to be changed (actually this program is not written rigidly, there may be inconsistent codes...)

**Comment by geoffreylitt** [comment]
Thanks for merging, and for creating this useful tool! :+1:

**Comment by motemen** [comment]
Thank you for the PR :smile:

**Pull request merged**